### PR TITLE
[TT-12125] Add fixed window rate limiter vars

### PR DESCRIPTION
**Pull request by titpetric**
https://tyktech.atlassian.net/browse/TT-12125

### CODE REVIEW

File: components/tyk-gateway/templates/deployment-gw-repset.yaml [hunk 1]
```diff
@@ -115,6 +115,8 @@ spec:
             value: "{{ .Values.global.oasValidateExamples }}"
           - name: TYK_GW_OAS_VALIDATE_SCHEMA_DEFAULTS
             value: "{{ .Values.global.oasValidateSchemaDefaults }}"
+          - name: TYK_GW_ENABLE_FIXED_WINDOW_RATE_LIMITER
+            value: "{{ .Values.gateway.enableFixedWindowRateLimiter }}"
```
Comment (komalsukhani) [inline components/tyk-gateway/templates/deployment-gw-repset.yaml:119]:
```suggestion
            value: "{{ .Values.gateway.enableFixedWindowRateLimiter | quote }}"
```

**Review by komalsukhani — CHANGES_REQUESTED** [review]

File: components/tyk-gateway/templates/deployment-gw-repset.yaml (hunk 1 above)
Comment (titpetric) [reply components/tyk-gateway/templates/deployment-gw-repset.yaml:119]: I'm not sure what this is supposed to do, seems redundant?

- boolean value
- already has quotes outside the template tag
- previous RL PR #238 also did not have quote

Comment (komalsukhani) [reply components/tyk-gateway/templates/deployment-gw-repset.yaml:119]: Ohh sorry. Yes It's redundant.

**Review by komalsukhani — APPROVED** [review]

**Comment by komalsukhani** [comment]
@titpetric Is this PR tested by your QA?

**Comment by komalsukhani** [comment]
@titpetric Unfortunately we don't have a testing guide. Our QA(Pranshu) can help your QA in testing it.

**Comment by komalsukhani** [comment]
@titpetric Is this PR tested? 
Do let us know once done, so we can merge it.

**Pull request merged**